client/src/components/Form/FormElement.vue

```diff
@@ -177,7 +177,7 @@ const isOptional = computed(() => !isRequired.value && attrs.value["optional"] !
 const formAlert = ref<string>();
 const alerts = computed(() => {
     return [formAlert.value, props.error, props.warning]
-        .filter((v) => v !== undefined && v !== null)
+        .filter((v) => v !== undefined && v !== null && v !== "")
```

Comment:
Why is there an empty string in one of the sources ? Isn't that would should be fixed ?

Reply:
It is returned in: https://github.com/galaxyproject/galaxy/blob/dev/client/src/components/Workflow/Editor/composables/useUniqueLabelError.ts#L13

Reply:
Can you change the ref to `const error = ref<string? | null>();` instead and then set error.value to `null` if there's no duplicate label ?